core/io/packet_peer_udp.h

```diff
@@ -46,8 +46,8 @@ class PacketPeerUDP : public PacketPeer {
 	};
 
 	RingBuffer<uint8_t> rb;
-	uint8_t recv_buffer[PACKET_BUFFER_SIZE];
-	uint8_t packet_buffer[PACKET_BUFFER_SIZE];
+	uint8_t recv_buffer[PACKET_BUFFER_SIZE] = { 0 };
+	uint8_t packet_buffer[PACKET_BUFFER_SIZE] = { 0 };
```

Comment:
Double check this, this may not be good idea, depending how big PACKET_BUFFER_SIZE is.